keyboards/ivy/keymaps/default/keymap.c

```diff
@@ -0,0 +1,44 @@
+#include QMK_KEYBOARD_H
+
+extern keymap_config_t keymap_config;
+
+// Each layer gets a name for readability, which is then used in the keymap matrix below.
+// The underscores don't mean anything - you can have a layer called STUFF or any other name.
+// Layer names don't all need to be of the same length, obviously, and you can also skip them
+// entirely and just use numbers.
+#define _L1 0
+
+#define _FUNC 15
```

Comment:
Is there a reason for setting this so high?  

Because by setting this to 15, it fills in all the intermediate layers with "NO"s, and unnecessarily bloats the firmware.